devtools/conda-envs/basic.yaml

```diff
@@ -28,18 +27,13 @@ dependencies:
 
   # openmmforcefields brings in the full toolkit; if that is ever re-packaged
   # this should be changed to openff-toolkit-base
-  - openff-toolkit >= 0.11.3
-  # argcomplete (brought in via BSE) pulls down totally incorrect builds of `importlib_metadata`,
-  - importlib_metadata >=4
-  - importlib-metadata >=4
```

Comment:
This is now included in recent builds of the toolkit and constraining `>= 0.12` will avoid pulling down older builds